mypy_django_plugin/django/context.py

```diff
@@ -375,30 +377,40 @@ def _resolve_field_from_parts(
         assert isinstance(field, (Field, ForeignObjectRel))
         return field
 
-    def resolve_lookup_into_field(
+    def solve_lookup_type(
         self, model_cls: Type[Model], lookup: str
-    ) -> Union["Field[Any, Any]", ForeignObjectRel]:
+    ) -> Optional[Tuple[Sequence[str], Sequence[str], Union[Expression, Literal[False]]]]:
         query = Query(model_cls)
```

Comment:
Note that Django itself never constructs a Query with an abstract model.

It would seem safer to test `model._meta.pk` and avoid constructing Query if it has no pk.

Reply:
Hmm, although we would construct Query() with an abstract model for other columns anyway. I guess it doesn't matter really.